src/puppetlabs/puppetdb/cli/benchmark.clj

```diff
@@ -428,7 +429,7 @@
   (close-buf! [this]
     (println "Shutting down AMQ...")
     (reset! queue-count 0)
-    (mq/stop-broker! broker)
+    (amq/stop-broker! broker)
```

Comment:
This is good, @rbrw is working on a PR now to remove this and move it to stockpile.